customer-terminate-service/app/src/test/java/se/inera/intyg/cts/infrastructure/persistence/JpaTerminationRepositoryTest.java

```diff
@@ -49,4 +61,32 @@ void shallReturnAllTerminationsWhenFindingAll() {
 
     assertEquals(numberOfTerminations, jpaTerminationRepository.findAll().size());
   }
+
+  @Test
+  void shallReturnTerminationsWithCorrectStatus() {
+    final var numberOfTerminations = 10;
+    for (int i = 0; i < numberOfTerminations; i++) {
+      inMemoryTerminationEntityRepository.save(
```

Comment:
In order to retrieve terminations we have to provide the data to be retrieved. Sure, if inMemory-implementation has an error it could cause the test to break, but the test verifies that the it reads terminations from the repository and returns them.